src/SoundOptionsModal.test.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { configure, mount } from 'enzyme';
+import { IntlProvider } from 'react-intl';
+import messages from './messages.json';
+import SoundOptionsModal from './SoundOptionsModal';
+import ToggleSwitch from './ToggleSwitch';
+
+configure({ adapter: new Adapter() });
+
+const defaultSoundOptionsModalProps = {
+    audioEnabled: true,
+    announcementsEnabled: true,
+    sonificationEnabled: true,
+    show: true
+};
+
+function createMountSoundOptionsModal(props) {
+    const mockOnCancel = jest.fn();
+    const mockOnSave = jest.fn();
+    const wrapper = mount(
+        React.createElement(
+            SoundOptionsModal,
+            Object.assign(
+                {},
+                defaultSoundOptionsModalProps,
+                {
+                    onCancel: mockOnCancel,
+                    onChangeSoundOptions: mockOnSave
+                },
+                props
+            )
+        ),
+        {
+            wrappingComponent: IntlProvider,
+            wrappingComponentProps: {
+                locale: 'en',
+                defaultLocale: 'en',
+                messages: messages.en
+            }
+        }
+    );
+
+    return {
+        wrapper,
+        mockOnCancel,
+        mockOnSave
+    };
+}
+
+function getAudioToggleSwitches(wrapper) {
+    return wrapper.find(ToggleSwitch);
+}
+
+function getCancelButton(wrapper) {
+    return wrapper.find('.ModalWithFooter__secondaryButton');
```

Comment:
This will need to be updated when C2LC-520 (new share modal) is merged.  I'd suggest adding a TODO in both functions.

Reply:
I think TODO on these functions are not needed as we will notice test case failure as soon as C2LC-520 gets merged in, but I added them.

Reply:
Thanks for humouring me.

Reply:
I find TODOs helpful even if I'm just immediately deleting them on the merge.